CHANGES.rst

```diff
@@ -62,6 +62,14 @@ Unreleased
   :meth:`.Subreddit.submit_image`, and :meth:`.Subreddit.submit_video` to
   support submitting as a live discussion (set to ``CHAT``).
 * Instances of :class:`.Trophy` can be compared for equality with each other.
+* PRAW now handles ratelimit errors returned as instances of
+  :class:`.RedditAPIException`.
+* :class:`.Reddit` has two new parameters, ``ratelimit_seconds`` and
```

Comment:
Let's keep this PR simple by only adding `ratelimit_seconds`. While there may end up being a need for retries, let's wait for feedback on that.